### ISSUE TITLE
Installation wizard - Chain setup completed 

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#188]
### Story
As a node operator, when the setup for a chain is finalised, I want to get on-screen confirmation, so that I can either start the process again for another chain, or go to the UI's homepage.
### Description
Here the node operator will be made aware that the monitoring set up was successful while being given the opportunity to either restart a new configuration process or go to PANIC's homepage.

Image 1
![installer setup completed](https://user-images.githubusercontent.com/89395527/166205207-fe8035f2-73ea-45b6-9ea6-f84de32bfa0f.PNG)

Image 2
![installer setup completed2](https://user-images.githubusercontent.com/89395527/166226622-e4dbb8ed-8672-411a-9b3e-04ede2c63833.PNG)
### Requirements
- Page to be displayed at displayed at /installation/feedback.
- Display vertically stacked and centred the Success logo, and the text "Success". (See Image 1)
- Display the text, "**_Sub-chain name (Base chain)_** has been just configured!". Note that the base chain and the sub-chain name would have already been selected and inputted by the node operator in earlier steps.
- Display the text, "Should you wish to edit the configuration of any sub-chain, please visit the **Home Page** and navigate to **Settings > Sub-chain**".
- After the node operator has s**uccessfuly gone through the installation process for one sub-chain**, introduce a table two columns  - **Base chain**, and **Sub-chain name**. This will allow the node operator to keep check on what chains have already been added to PANIC for monitoring.** (See Image 2)
- The base chain column will consist of 3 possible entries - Substrate, Cosmos-SDK, and Chainlink.
- The sub-chain column will consist of the names given to the various chains by the node operator in the Blockchain Setup phase.
- Display  a button titled, Configure another chain. This will take the node operator back to the 1st step - Blockchain Setup. (See Image 1)
- Display  a button titled, Home page. This will be used to close off the installer and the node operator is taken to PANIC's homepage
### Blocked by
 Not dependent on another ticket to be tackled.
### Acceptance Criteria
**Scenario**: Node operator confirms his configuration
**When**: The node operator confirms his config setup by clicking on the Finish button (see #186)
**Then**: He is presented with the Chain setup completed page
**And**: If the node operator clicks on the Configure another chain button
**Then**: He is taken to the 1st step (Blockchain Setup) 
**And**: If the node operator clicks on the "Home Page" button
**Then**: He is taken to PANIC's homepage - i.e. Dashboard overview
**And**: If the chain just configured **is not** the node operator's first
**Then**: Display a table that shows the previously setup chains including their base chain and sub-chain name
**And**: If the chain just configured **is** the node operator's first
**Then**: Do not display the table

**Comment by crsfalzon** [comment]
Data table can be omitted
